sdk/python/kfp/cli/diagnose_me_cli.py

```diff
@@ -27,7 +28,7 @@ def diagnose_me():
 @click.option(
     '--namespace',
     type=Text,
-    help='Namespace to use for Kubernatees cluster. It will use environment default if not specified.'
+    help='Namespace to use for Kubernetes cluster. It will use environment default if not specified.'
```

Comment:
Can you clarify whether it is a default value specified in env variable or the `default` namespace?

Reply:
The help message is in correct, if not specified --all-namespaces will be used. updated accordingly.